### ISSUE TITLE
treeify fails with very very long account names

### ISSUE THREAD

**Issue by blais** [beancount/beancount#14]
**[Original report](https://bitbucket.org/blais/beancount/issue/14) by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

2. I like how you split treeify and bean-report, however, this is causing it to not work for me. I have some rather long account names, and as a result the number of spaces between the account name and the amount isn't enough for treeify to known where to split.

As a workaround I'm using (hooray for Unix utils!):

bean-report ledger.bean bal | sed -e 's/\s\s*/\t/' | expand -t 50 | treeify

**Comment by blais** [comment]
**Original comment by Justus Pendleton (Bitbucket: [hoostus](https://bitbucket.org/hoostus), GitHub: [hoostus](https://github.com/hoostus)).**

----------------------------------------

I think this can closed. (Maybe bean-report or treeify has been changed in some way since this bug was filed in early 2015?)

I tried using a 170+ character account name and treeify had no problems knowing where to split things. If this is still a problem and someone can provide a more concrete example of how to reproduce it, I'm happy to investigate more, though.
